packages/dart/npt_flutter/lib/features/profile_list/view/profile_list_view.dart

```diff
@@ -84,15 +84,12 @@ class ProfileListView extends StatelessWidget {
                                     child: ListView.builder(
                                       itemCount: state.profiles.length,
                                       itemBuilder: (context, index) {
-                                        if (state.profiles.isEmpty) {
```

Comment:
This if shouldn't be here and prevents profiles from loading when they should be loading. Left over from a refactor